### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by nguyenminhduong**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="com.example.framgia.soundclound_01.MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"/>
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: hard code

File: app/src/main/java/com/example/framgia/soundclound_01/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.example.framgia.soundclound_01;
```
Comment (DoanVanToan) [inline app/src/main/java/com/example/framgia/soundclound_01/MainActivity.java]: chưa chia package theo convention.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<manifest package="com.example.framgia.soundclound_01"
+          xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity android:name=".MainActivity">
```
Comment (vtabk2) [inline app/src/main/AndroidManifest.xml]: chưa thấy MainActivity

**Comment by vtabk2** [comment]
ok

File: .gitignore [hunk 4]
```diff
@@ -35,6 +35,7 @@ captures/
 # Intellij
 *.iml
 .idea/workspace.xml
+.idea/*
 
 # Keystore files
-*.jks
+*.jks
```
Comment (DoanVanToan) [inline .gitignore]: chưa format code

**Pull request merged**